.github/workflows/build_windows.yml

```diff
@@ -4,17 +4,15 @@ on:
   pull_request:
     paths:
     - .github/workflows/build_windows.yml
-    - ansible/DockerFile.Windows2016_Base
-    - ansible/Dockerfile.Windows2016_VS2013
-    - ansible/Dockerfile.Windows2016_VS2017
+    - ansible/docker/DockerFile.Windows2016_Base
+    - ansible/docker/Dockerfile.Windows2016_VS2017
     branches:         
     - master
   push:
     paths:
     - .github/workflows/build_windows.yml
-    - ansible/DockerFile.Windows2016_Base
-    - ansible/Dockerfile.Windows2016_VS2013
-    - ansible/Dockerfile.Windows2016_VS2017
```

Comment:
removing 2013 on purpose?

Reply:
yeah, 2013 never existed, forgot to remove it from the file when I originally added the action